tests/app/clients/test_zendesk_sell.py

```diff
@@ -372,17 +416,22 @@ def test_create_service_or_go_live_deal_fail_contact_exists(
             contact_delete_mock.assert_not_called()
 
 
+@pytest.mark.parametrize('existing_contact_id', [None, '1'])
 def test_send_create_service(
         notify_api: Flask,
         sample_service: Service,
-        mocker: MockFixture):
+        mocker: MockFixture,
+        existing_contact_id: Optional[str]):
 
-    contact_id = 1
+    contact_id = existing_contact_id or '1'
```

Comment:
Curious about this because `contact_id` will always be `1` since you pass `None`

Reply:
my mistake they should be different, changed.